fuel/datasets/hdf5.py

```diff
@@ -542,8 +542,11 @@ def get_data(self, state=None, request=None):
             data, shapes = self._out_of_memory_get_data(state, request)
         for i in range(len(data)):
```

Comment:
equizip would be more defensive

On Tue, Jan 26, 2016, 4:06 PM dmitriy-serdyuk <notifications@github.com>
wrote:

> In fuel/datasets/hdf5.py
> <https://github.com/mila-udem/fuel/pull/289#discussion_r50899917>:
>
> > @@ -542,8 +542,11 @@ def get_data(self, state=None, request=None):
> >              data, shapes = self._out_of_memory_get_data(state, request)
> >          for i in range(len(data)):
>
> for data_point, shape in zip(data, shapes)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/fuel/pull/289/files#r50899917>.
>